src/SearchInput/SearchInput.tsx

```diff
@@ -11,23 +12,39 @@ import { CommonFieldProps } from '../fields/commonFieldTypes'
 import { Input, StyledFrontIcon } from '../fields/components/CommonInput'
 import { useUniqueId } from '../utils/id'
 import { useControllableState } from '../utils/useControlledState'
-import { SearchOptions } from './SearchOptions'
+import { SearchOptions } from './components/SearchOptions'
+import Fuse from 'fuse.js'
 
 export type SearchInputItem = {
   label: string
   value: string
 }
 
 export interface SearchInputProps extends CommonFieldProps {
+  /** Optional input className */
   name?: string
+  /**  Optional placeholder text */
   placeholder?: string
+  /**  List of input items to search on*/
   searchList: SearchInputItem[]
+  /**  callback to handle found item click */
   onFound: (element: string) => void
+  /**  optional callback to run when no results found */
+  onNotFound?: () => void
+  /**  optional Component to render when no results found */
+  notFoundComponent?: ReactNode
+  /**  optional boolean to show search icon */
   showIcon?: boolean
+  /**  Optional callback to run on blur */
   onBlur?: (e: FocusEvent<HTMLInputElement>) => void
+  /**  Optional default value for input */
   value?: string
+  /**  Optional boolean to move results to a realtive position */
   resultsRelativePosition?: boolean
+  /**  optional boolean to add border to results */
   resultsBorder?: boolean
+  /** optional boolean to enable fuzzy search via fuse.js */
+  useFuzzySearch?: boolean
```

Comment:
Suggestion: maybe calling it `enableFuzzySearch` would be better as it avoid using the `use` keyword reserved for hooks?